src/Microsoft.Fx.Portability.Offline/OfflineApiPortService.cs

```diff
@@ -57,6 +58,13 @@ public Task<ServiceResponse<AnalyzeResponse>> SendAnalysisAsync(AnalyzeRequest a
         public Task<ServiceResponse<IEnumerable<ReportingResultWithFormat>>> SendAnalysisAsync(AnalyzeRequest a, IEnumerable<string> formats)
         {
             var response = _requestAnalyzer.AnalyzeRequest(a, Guid.NewGuid().ToString());
+
+            if(formats == null || !formats.Any())
```

Comment:
Change to `if (!formats?.Any() ?? true) { // do stuff when there are no formats. } `